runtime/sema/return_info.go

```diff
@@ -19,18 +19,18 @@
 package sema
 
 type ReturnInfo struct {
-	// MaybeJumpedOrReturned indicates that the (branch of) the function
-	// contains a potentially taken return, break, or continue statement
-	MaybeJumpedOrReturned bool
-	DefinitelyReturned    bool
-	DefinitelyHalted      bool
-	DefinitelyJumped      bool
+	// MaybeReturned indicates that (a branch of) the function
```

Comment:
Thank you for fixing the typo here (double "the"). Sorry to be so pernickety, but maybe use "the" branch in the parentheses, as this return info is not for some branch of the function, but "this particular one" / "the current one".
Just worries "a branch" could be misinterpreted to mean "some branch" / "another branch" in the function.

```suggestion
	// MaybeReturned indicates that (the branch of) the function
```